dll/win32/syssetup/wizard.c

```diff
@@ -2831,6 +2831,53 @@ ProcessUnattendSection(
     }
 
     RegCloseKey(hKey);
+
+    if (SetupFindFirstLineW(pSetupData->hSetupInf,
```

Comment:
Indent seems inconsistent wrt. the rest of the code in this file.

Reply:
Indeed, but I don't feel like manually spacing that stuff out